apps/scoutgame/components/matchup/components/RegistrationButton.tsx

```diff
@@ -73,7 +73,13 @@ export function RegistrationButton({ registered, week }: { registered: boolean;
         disabled={registered}
         variant='contained'
         color='secondary'
-        endIcon={registered ? <CheckCircleIcon color='inherit' /> : <PointsIcon color='inherit' />}
+        endIcon={
+          registered ? (
+            <CheckCircleIcon color='inherit' />
+          ) : (
+            <Image src='/images/dev-token-logo.png' alt='DEV token' width={18} height={18} />
```

Comment:
TODO later: we should make this a component so we're not hardcoding the image path everywhere